### PR TITLE
Death event modification

### PR DESCRIPTION
**Pull request by matvejs16**
I made some changes to get more information about player death.

### CODE REVIEW

File: lib/plugins/health.js [hunk 1]
```diff
@@ -22,12 +22,23 @@ function inject (bot) {
     if (bot.health <= 0) {
       if (bot.isAlive) {
         bot.isAlive = false
-        bot.emit('death')
```
Comment (extremeheat) [inline lib/plugins/health.js:25]: What about other versions?

Comment (matvejs16) [reply lib/plugins/health.js:25]: [Added support for <= 1.16.5 versions](https://github.com/PrismarineJS/mineflayer/pull/3026/commits/72bb2dd5f3bb49f219af7b8a0c867d076370dbe3)
- [x] 1.19.3
- [x] 1.19.2
- [x] 1.19
- [x] 1.18.2
- [x] 1.17.1
- [x] 1.16.5
- [x] 1.15.2
- [x] 1.14.4
- [x] 1.13.2
- [x] 1.12.2
- [x] 1.11.2
- [x] 1.10.2
- [x] 1.9.4
- [ ] 1.8.8

Got some problem when tried to connect to localhost singleplayer server on 1.8.8 (That's why 1.8.8 is unchecked)
```
const bot = mineflayer.createBot({
 host: 'localhost',
 port: 64320,
 username: botData.username,
 viewDistance: 'tiny'
})
```

Error: 
```
Error: connect ECONNREFUSED ::1:64320
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '::1',
  port: 64320
}

node:internal/event_target:1010
  process.nextTick(() => { throw err; });
                           ^
Error: connect ECONNREFUSED ::1:64320
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16)
Emitted 'error' event on PoolWorker instance at:
    at [kOnErrorMessage] (node:internal/worker:290:10)
    at [kOnMessage] (node:internal/worker:301:37)
    at MessagePort.<anonymous> (node:internal/worker:202:57)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:735:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '::1',
  port: 64320
}
```

**Comment by matvejs16** [comment]
Any updates in this PR?

**Comment by rom1504** [comment]
please re-open if you want to finish it

**Pull request closed without merging**